bundle/edu.gemini.oodb.too.url/src/main/java/edu/gemini/spdb/rapidtoo/impl/TooUpdateServiceImpl.java

```diff
@@ -237,10 +237,9 @@ private void _setTarget(ISPObservation obs, TooUpdate update)  {
         TargetEnvironment targetEnv = targetObsComp.getTargetEnvironment();
         SPTarget base = targetEnv.getBase();
 
-        base.getTarget().setName(tooTarget.getName());
-        base.getTarget().getRa().setAs(tooTarget.getRa(), CoordinateParam.Units.DEGREES);
-        base.getTarget().getDec().setAs(tooTarget.getDec(), CoordinateParam.Units.DEGREES);
-        base.getTarget().setMagnitudes(tooTarget.getMagnitudes());
+        base.setName(tooTarget.getName());
+        base.setRaDecDegrees(tooTarget.getRa(), tooTarget.getDec());
+        base.setMagnitudes(tooTarget.getMagnitudes());
         base.notifyOfGenericUpdate();
```

Comment:
Before we would call `notifyOfGenericUpdate` once at the end of a string of updates like this.  Now there will be events per mutation?  I think that's probably fine but it is a difference in behavior and I know a lot happens when targets are updated.  This is just a transition step though until we simply swap out the immutable target once it is properly configured?

At any rate, it seems like we shouldn't not need the last `notifyOfGenericUpdate` call here if all the individual mutations are generating events anyway.